.github/workflows/npm_release.yml

```diff
@@ -0,0 +1,62 @@
+name: Release npm package
+on:
+  workflow_dispatch:
+    inputs:
+      version-bump:
+        description: The scale of the version bump required for semver compatibility
+        required: true
+        default: patch
+        type: choice
+        options:
+          - patch
+          - minor
+          - major
+concurrency: release
+jobs:
+  release:
+    name: "Release & Publish"
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./app
+    steps:
+      - name: 🧮 Checkout code
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.ELEMENT_BOT_TOKEN }}
+
+      - name: 🔧 Set up node environment
+        uses: actions/setup-node@v3
+        with:
+          cache: "yarn"
+
+      - name: 🛠️ Setup
+        run: yarn install --pure-lockfile
+
+      - name: 👊 Bump version
+        run: |
+          yarn version --no-git-tag-version --${{ github.event.inputs.version-bump }}
+          git config --global user.name 'ElementRobot'
+          git config --global user.email 'releases@riot.im'
+          git commit -am "${{ github.event.inputs.version-bump }} version bump"
+          git push
+      - name: ⚙️ Build
+        run: |
+          yarn build
```

Comment:
You don't need this as your build script is identical to your `prepare` script which is called by publishing.